mctreesearch4j/build.gradle

```diff
@@ -37,14 +44,25 @@ publishing {
         }
     }
 }
+*/
 
 jar {
     manifest {
-        attributes["Main-Class"] = "com.mcts.MDP"
+        attributes["Main-Class"] = "ca.aqtech.mctreesearch4j.MDP"
     }
     // This line of code recursively collects and copies all of a project's files
     // and adds them to the JAR itself. One can extend this task, to skip certain
     // files or particular types at will
     from { configurations.compileClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
 }
 
+apply from: "${rootDir}/scripts/publish-root.gradle"
+
+ext {
+    PUBLISH_GROUP_ID = 'ca.aqtech'
+    PUBLISH_VERSION = '1.0-SNAPSHOT'
```

Comment:
Just to confirm, this is just the dev version right? What will be the RTM version?

Reply:
I'm just trying to get this working for now, is 1.0.0 good? Should we start with 0.0.1-beta?

Reply:
Start with 0.0.1-beta then. Reserve 1.0.0 for something we actually mean to ship as a first version.